persistent-sqlite/ChangeLog.md

```diff
@@ -4,6 +4,8 @@
 
 * [#1060](https://github.com/yesodweb/persistent/pull/1060)
   * The QuasiQuoter now supports `OnDelete` and `OnUpdate` cascade options.
+* [#1069](https://github.com/yesodweb/persistent/pull/1070)
```

Comment:
Is this 1069/1070 mismatch intentional?

Reply:
(This also applies to the other changelog files)

Reply:
I think I somehow came to the conclusion that the first is the issue number while the second is the PR, but I'll change it to the PR everywhere.